root/utility/tableColumns.js

```diff
@@ -143,6 +143,29 @@ export function defineEntityColumn<D>(
   };
 }
 
+export function defineTextColumn<D>(
```

Comment:
This could probably be used for the label code too? (Maybe others)

Reply:
By making the passed getText something like `entity => formatLabelCode(entity.label_code)` ?

Reply:
Looks like the current label code column handles empty values, so something like `entity => entity.label_code ? formatLabelCode(entity.label_code) : ''` seems more equivalent to what you have, but yeah.

Reply:
Ok, done. Also changed order key code to label_code to avoid adding a new argument (and because it didn't make sense anyway)